string name
string type
string[] remap_source_args
string[] remap_target_args
---
bool success
